CHANGELOG.md

```diff
@@ -101,8 +101,8 @@ Every module contains its own CHANGELOG.md. Please refer to the module you are i
 
 ### API Breaking Changes
 
+* (types) [#19792](https://github.com/cosmos/cosmos-sdk/pull/19792) In `MsgSimulatorFn` `sdk.Context` and `[]Account` arguments are replaced for an `address.Codec`. It also returns an error.
```

Comment:
This warrant a mention in the UPGRADING.md as well. Have you doubled checked as well on sourcegraph how the api is used to be sure 100% no user depends on it?

Reply:
I am quite sure no one uses sdk context but not too sure about the random accounts

Reply:
This is the only one I could find. https://github.com/EmpowerPlastic/empowerchain/blob/03668adb20f0e92dc80667c0c6735bf3bee968c2/chain/x/plasticcredit/simulation/proposals.go#L39-L51

But may be others actually 🤔

Reply:
Okay, so that's enough to be the account argument imho.